libraries/AP_HAL_Linux/RCOutput_Multi.cpp

```diff
@@ -0,0 +1,186 @@
+#include "RCOutput_Multi.h"
+#include <stdio.h>
+#include <AP_HAL/AP_HAL.h>
+
+#define MIN(a,b) ((a)<(b)?(a):(b))
+
+#ifdef DEBUG
+#define debug(fmt, args ...) do { fprintf(stderr,"[RCOutput_Multi]: %s:%d: " fmt "\n", __FUNCTION__, __LINE__, ## args); } while (0)
+#else
+#define debug(fmt, args ...)
+#endif
+
+extern const AP_HAL::HAL& hal;
+
+using namespace Linux;
+
+RCOutput_Multi::~RCOutput_Multi()
+{
+}
+
+void RCOutput_Multi::init()
+{
+    for( unsigned g = 0; g < n_out_groups; ++g )
+        {
+        std::get<0>(out_groups[g]).init();
+        }
+}
+
+bool RCOutput_Multi::resolve_channel(uint8_t ch, unsigned & gid, unsigned &cid)
```

Comment:
tighten the type from unsigned to something from the standard integer set (eg `uint8_t`)

Reply:
done